test/memory_tools/test_memory_tools.cpp

```diff
@@ -20,22 +20,22 @@
  */
 TEST(TestMemoryTools, test_allocation_checking_tools) {
   size_t unexpected_mallocs = 0;
-  auto on_unexpected_malloc = (
-    [&unexpected_mallocs]() {
+  auto on_unexpected_malloc = [&unexpected_mallocs]()
+    {
```

Comment:
Same comment as this one: https://github.com/ros2/rcl/pull/215#discussion_r171133823

Reply:
Updated the same way.